client/js/render.js

```diff
@@ -305,18 +315,24 @@ function loadMoreHistory(entries) {
 	});
 }
 
-sidebar.on("click", ".collapse-network", (e) => {
-	const collapseButton = $(e.target);
+sidebar.on("click", "button.collapse-network", (e) => collapseNetwork($(e.target)));
 
+function collapseNetwork(target) {
+	const collapseButton = target.closest(".collapse-network");
+	const networks = JSON.parse(storage.get("thelounge.networks.collapsed")) || {};
+	const networkuuid = collapseButton.closest(".network").attr("data-uuid");
 	collapseButton.closest(".network").toggleClass("collapsed");
 
 	if (collapseButton.attr("aria-expanded") === "true") {
 		collapseButton.attr("aria-expanded", false);
 		collapseButton.attr("aria-label", "Expand");
+		networks[networkuuid] = true;
 	} else {
 		collapseButton.attr("aria-expanded", true);
 		collapseButton.attr("aria-label", "Collapse");
+		networks[networkuuid] = false;
```

Comment:
`delete networks[networkuuid]` instead?

Reply:
I thought about that. But logically, does this make more sense to devs looking?

Reply:
I believe so. You're storing a map of only collapsed networks, not all of their states.

Reply:
No I get that, but my point wsas that `networks[networkuuid] = false` is more obvious what is happening than `delete networks[networkuuid]` I thought, but sure.